src/lib/components/schedule/schedules-time-view.svelte

```diff
@@ -38,43 +14,14 @@
   <p>
     {translate('schedules.time-view-description')}
   </p>
-  <div class="flex flex-col items-center gap-2 lg:flex-row">
-    <Input
-      id="hour-time"
-      class="w-24"
-      label={translate('common.hour-abbreviated')}
-      labelHidden
-      bind:value={_hour}
-      placeholder="00"
-      suffix={translate('common.hour-abbreviated')}
-      maxLength={2}
-      hideCount
-      error={error(_hour, 12)}
-    />
-    <div class="hidden lg:block">:</div>
-    <Input
-      id="minute-time"
-      class="w-24"
-      label={translate('common.minutes-abbreviated')}
-      labelHidden
-      bind:value={minute}
-      placeholder="00"
-      suffix={translate('common.minutes-abbreviated')}
-      maxLength={2}
-      hideCount
-      error={error(minute, 59)}
-    />
-    <div class="ml-2">
-      <ToggleButtons>
-        <ToggleButton active={time === 'AM'} on:click={() => (time = 'AM')}
-          >{translate('common.ante-meridiem')}</ToggleButton
-        >
-        <ToggleButton active={time === 'PM'} on:click={() => (time = 'PM')}
-          >{translate('common.post-meridiem')}</ToggleButton
-        >
-      </ToggleButtons>
-    </div>
-  </div>
+
+  <TimePicker
+    bind:hour
+    bind:minute
+    twelveHourClock={false}
```

Comment:
@Alex-Tideman updated to use 24 hour time instead ✅

Reply:
Awesome!